README.md

```diff
@@ -103,15 +119,34 @@ by adding -DUSE_CPU_ONLY=1 to the cmake command line.
     # Install build depedencies
     sudo apt install libboost-dev libboost-program-options-dev libboost-filesystem-dev opencl-headers ocl-icd-libopencl1 ocl-icd-opencl-dev zlib1g-dev
 
-    # Use stand alone directory to keep source dir clean
+    # Use a stand alone build directory to keep source dir clean
     mkdir build && cd build
+    
+    # Compile leelaz and autogtp in build subdirectory with cmake
     cmake ..
     cmake --build .
+    
+    # Optionnal : if you want to test if your build has no issue
     ./tests
+
+### Then, if you just want to play right now (not contribute), run leelaz :
+
+    # Use the command below to download leelaz and run leelaz in play mode
     curl -O https://zero.sjeng.org/best-network
     ./leelaz --weights best-network
+    
+### Else, if you want to help (to contribute), run instead autogtp :
+
+    # Copy autogtp and leelaz binaries from build subdirectory to autogtp subdirectory
```

Comment:
Nit: I don't believe these steps are necessary. You can technically run ./autogtp/autogtp from the build dir and everything works fine.

Reply:
i think the philosophy of the build folder was to build sources (in the past it was the src folder instead), and the autogtp folder is intended to contain autogtp

if you think we dont need autogtp folder, we should just delete it i think

Reply:
If we can simplify the instructions we should do so, I don't understand what this would have to do with there being an autogtp folder or not.